src/main/features/offer/views/settled.njk

```diff
@@ -0,0 +1,32 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+
+{% set title = t('Money claim | Claim settled - confirmation page') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="govuk-box-highlight">
+        <h1 class="bold-large">{{ t('You’ve settled out of court') }}</h1>
```

Comment:
shouldn't this be heading-large not bold-large class?
REQ

Reply:
Done